doc/interfaces_overview.md

```diff
@@ -59,14 +59,15 @@ sys.path.append(
 import grass.script as gs
 
 # Create a new project
-gs.create_project(path=grassdata, name=project_name, epsg="3358", overwrite=False)
+gs.create_project(path="path/to/my_project", epsg="3358")
 
 # Initialize the GRASS session
-with gs.setup.init(Path("grassdata/project_name")) as session:
+with gs.init("path/to/my_project") as session:
```

Comment:
Same as above. Here, perhaps, we see that the current state is not consistent since create_project is now usable, the separation of _setup_ does not make that much sense. Right now, however:

```suggestion
with gs.setup.init("path/to/my_project") as session:
```